### PR TITLE
playground card added next to image hub. 

### PR DESCRIPTION
**Pull request by EdwinDij**
the card link to:[/cloud-native-management/playground]

**Description**

Create a new project card for the Meshery Playground and link it to layer5.io/cloud-native-management/playground. This card includes the Meshery logo and the title "Meshery Playground". The new page has been created under issue #4151.

**Notes for Reviewers**

Please review and approve this pull request. The changes have been made as requested and the new project card has been added to layer5.io/projects. I have signed my commits in accordance with the Layer5 contribution guidelines.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8a905a355c3b6c2ccd54caea7fc39e72a80c927f at: https://645b855047bc100088b405f5--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit bbaa944c0c73459e47dfc38c2297676f1c6aaadc at: https://645d00b5a419a823e2ff1db4--layer5.netlify.app

File: src/sections/Projects/Project-grid/index.js [hunk 1]
```diff
@@ -114,6 +114,12 @@ const ProjectPage = () => {
                 </Row>
               </div>
             </Link>
+            <Link to="/cloud-native-management/playground"className="project__card ten">
```
Comment (Nikhil-Ladha) [inline src/sections/Projects/Project-grid/index.js:117]: This link is not yet active, let's hold off this PR until #4184 is merged

**Comment by ayushthe1** [comment]
@EdwinDij   Let's discuss this on websites call. Add this as an agenda item in meeting minutes if you would :).
I'll add this item for you if it's ok.

**Comment by EdwinDij** [comment]
@ayushthe1 yes it's ok for me

**Comment by leecalcote** [comment]
@EdwinDij will you have a look at and resolve the merge conflict(s)?

**Comment by EdwinDij** [comment]
@leecalcote Yes I will fix it this afternoon. But it seems that playground were deleted from the code without his css. Is it ok if I put it back?

**Comment by leecalcote** [comment]
@Shivam-AfA

**Comment by Shivam-AfA** [comment]
@EdwinDij What issue are you facing here?

**Pull request closed without merging**